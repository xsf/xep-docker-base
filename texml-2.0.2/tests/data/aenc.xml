<?xml version="1.0"?>
<TeXML>Hello, [я]!</TeXML>
