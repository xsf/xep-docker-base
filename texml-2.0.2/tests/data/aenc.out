Hello, [^^d1^^8f]!
